conf/streamline-dev.yaml

```diff
@@ -120,6 +126,7 @@ logging:
   loggers:
     # Sets the level for 'com.example.app' to DEBUG.
     com.hortonworks.streamline: DEBUG
+    org.apache.tomcat.jdbc.pool.PooledConnection: ERROR
```

Comment:
is it related to the patch ? Why PooledConnection is separately handled? Root level can be set to ERROR instead.